.github/workflows/prerelease.yml

```
@@ -13,40 +13,50 @@ env:
 
 jobs:
   prerelease:
+    # only run this workflow for pydata owned repositories (avoid forks)
+    if: github.repository_owner == 'pydata'
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

so IIUC, we're running the pre-release distribution test on 5 python versions, but building wheel only on the py3.9 job.  Seems like overkill; here again I'd lean toward:

```suggestion
        python-version: ["3.9", "3.13"]
```

But maybe you have reasons I'm not thinking of that we should do this on all py versions?